MicroAOD/interface/CutBasedElectronObjectSelector.h

```diff
@@ -1,13 +1,62 @@
 #ifndef flashgg_CutBasedElectronObjectSelector_h
```

Comment:
Is this code used for the differential analysis or for the T&P?

Reply:
Comes from here commit https://github.com/threiten/flashgg/commit/59f86a37aea8a194bfc2500f44fdd9aab647a999#diff-5d1343eef6f9b26821aff25349a9c9f3
So used for differentials.

Reply:
ok